java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/websocket/GenericMqttWebSocketFactory.java

```diff
@@ -0,0 +1,34 @@
+package com.cumulocity.generic.mqtt.client.websocket;
+
+import com.cumulocity.generic.mqtt.client.*;
+import com.cumulocity.generic.mqtt.client.model.GenericMqttMessage;
+import com.cumulocity.sdk.client.messaging.notifications.TokenApi;
+
+public class GenericMqttWebSocketFactory implements GenericMqttFactory {
+    private final String webSocketBaseUrl;
+    private final TokenApi tokenApi;
+
+    /**
```

Comment:
suggestion: generally like this but maybe some minor changes to first sentence such as \<code\> tags around GenericMqttWebSocketFactory  and change '... and sets it to connect to the specified URI' to '... set to connect to the specified URI'

Reply:
Not relevant after recent changes